### DISCUSSION TITLE
I put my fangame "Squirt Goes to France" on mobile

### DISCUSSION THREAD

**Discussion by Squidward578** [TrianglyRU/OrbinautFramework#54] [Show and tell]
![image](https://github.com/user-attachments/assets/9bfba131-bd3d-48f0-95a0-c374eb697f7f)
the water shader doesn't work but it's still pretty fun to play with on mobile
![image](https://github.com/user-attachments/assets/06a75c8c-87e0-4a12-8c0a-210a5f7278d7)

**Comment by Otukuoi** [comment]
is this an inside joke or

**Reply by Squidward578** [reply]
i couldnt think of a better name sorry
